.github/CODEOWNERS

```diff
@@ -1,4 +1,4 @@
-* @rajsite @atmgrifter00 @cameronwaterman
+* @rajsite @cameronwaterman
```

Comment:
@atmgrifter00 don't forget to also update: https://github.com/ni/grafana-plugins/blob/135c1794317042c3446eede1f4caaf3be01a1caa/CODEOWNERS#L1

Reply:
And https://github.com/ni/grafana/blob/94c58ce83b586adbe91520c489c7b1df806cb921/.github/CODEOWNERS#L1